### PR TITLE
feat: add manage alerts button

### PR DESCRIPTION
**Pull request by MounirDhahri**
This PR resolves [ONYX-303] 

### Description
This PR adds manage alerts button when a user did bid on an artwork but lost it. This is because we automatically create alerts when a user loses a bid.

Group 1 (3)

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- Add manage alerts button - mounir

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[ONYX-303]: https://artsyproduct.atlassian.net/browse/ONYX-303?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Cross-platform user-facing changes (Add manage alerts button - mounir)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 0c42b20f5b7b41bcde4cf1bedcb2da4215840516
</p>

File: src/app/Scenes/Artwork/ArtworkAuctionCreateAlertHeader.tsx [hunk 1]
```diff
@@ -51,22 +56,38 @@ export const ArtworkAuctionCreateAlertHeader: FC<ArtworkAuctionCreateAlertHeader
...
+            size="large"
+            variant="outline"
+            haptic
+            onPress={() => {
+              navigate("/my-profile/saved-search-alerts")
+            }}
+            icon={<BellIcon fill="black100" />}
+            flex={1}
+          >
+            Manage Alerts
```
Comment (tam-kis) [inline src/app/Scenes/Artwork/ArtworkAuctionCreateAlertHeader.tsx:77]: The copy should be "Manage your alerts" - has something changed in the meantime?

Comment (dariakoko) [reply src/app/Scenes/Artwork/ArtworkAuctionCreateAlertHeader.tsx:77]: Not `Manage your Alerts`?
With the capital A

**Review by dariakoko — APPROVED** [review]
LGTM

File: src/app/Scenes/Artwork/ArtworkAuctionCreateAlertHeader.tsx (hunk 1 above)
Comment (tam-kis) [reply src/app/Scenes/Artwork/ArtworkAuctionCreateAlertHeader.tsx:77]: Without the capital `A`, it's the same in Force.

Comment (tam-kis) [reply src/app/Scenes/Artwork/ArtworkAuctionCreateAlertHeader.tsx:77]: Thought it would make sense to align it with the other cases...

Comment (MounirDhahri) [reply src/app/Scenes/Artwork/ArtworkAuctionCreateAlertHeader.tsx:77]: nice catch, will update

**Pull request merged**